packages/augur-ui/src/modules/modal/components/modal-global-chat.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { lazy, Suspense, useState } from 'react';
+import { Close } from 'modules/common/icons';
+import Styles from 'modules/modal/modal.styles.less';
+import classNames from 'classnames';
+import { use3box } from 'utils/use-3box';
+import { useAppStatusStore } from 'modules/app/store/app-status';
+import { useAppStatus } from 'modules/app/store/app-status-hooks';
+
+const ThreeBoxChat = lazy(() =>
+  import('modules/global-chat/components/three-box-chat')
+);
+
+export interface ModalGlobalChatProps {
+  closeModal: Function;
+  provider: any;
+  whichChatPlugin: string;
+  isLogged: boolean;
+}
+
+export const ModalGlobalChat = () => {
+  let { isLogged, loginAccount, env, initialized3box, actions: { closeModal } } = useAppStatusStore();
+  const signer = loginAccount.meta?.signer;
+  const provider = signer ? signer.provider?._web3Provider : false;
+  let { setInitialized3Box } = useAppStatus();
```

Comment:
same thing as the previous file, never use `useAppStatus` outside of the time it's defined in appStatus component provider. add `setInitilized3Box` as part of the things imported in the `actions` section of `useAppStatusStore()`